asn1ate/sema.py

```diff
@@ -719,6 +721,19 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self, elements):
+	self.module2symbols = { }
+	for i in range(0, len(elements), 2):
+		self.module2symbols [elements [i+1].elements[0]] = set(elements [i])
+
+class Exports(SemaNode):
```

Comment:
Did that, although it is derived from the semantical model so the syntax may change but not the meaning:

Input:
```
EXPORTS ALL, x;
```

Output:
```
EXPORTS x,ALL;
```